apps/remix-ide/src/app/udapp/run-tab.js

```diff
@@ -169,7 +169,7 @@ export class RunTab extends ViewPlugin {
     await addProvider('injected-optimism-provider', 'L2 - Optimism Provider', true, false)
     await addProvider('injected-arbitrum-one-provider', 'L2 - Arbitrum One Provider', true, false)
   
-    await addProvider('walletconnect', 'Wallet Connect', false, false)
+    await addProvider('walletconnect', 'WalletConnect', false, false)
```

Comment:
this second parameter is a plugin name (see above) I think this should be `Wallet Connect`

Reply:
See Andy's comment.